config/test/kubemacpool.yaml

```diff
@@ -293,7 +293,7 @@ spec:
           value: 0.0.0.0
         - name: HEALTH_PROBE_PORT
           value: "9440"
-        image: registry:5000/kubevirt/kubemacpool:latest
+        image: quay.io/kubevirt/kubemacpool:latest
```

Comment:
Dont we need to take the image from the local registry container so cluster-sync will affect?

Reply:
This is not to be like this... argg will fix thanks.